package.json

```diff
@@ -35,15 +35,16 @@
     "eslint-config-fxa": "2.1.0",
     "fxa-conventional-changelog": "1.1.0",
     "grunt-bump": "0.8.0",
-    "grunt-conventional-changelog": "6.1.0",
+    "grunt-conventional-changelog": "5.0.0",
     "grunt-copyright": "0.3.0",
-    "grunt-eslint": "19.0.0",
+    "grunt-eslint": "17.3.1",
     "grunt-jscs": "3.0.1",
     "grunt-mocha-test": "0.13.2",
     "grunt-nodemon": "0.4.2",
     "grunt-nsp": "2.3.1",
     "grunt-todo": "0.5.0",
     "jscs-jsdoc": "2.0.0",
+    "mocha": "^3.1.0",
```

Comment:
I think it somehow moved to a peer-dependency with one of the updates; in any case, tests refused to run without me explicitly installing it.